ipynb/balance_trees.ipynb

```diff
@@ -0,0 +1,488 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# __Balance Trees__"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "\n",
+    "\n",
+    "We will introduce the concept of balances here.  Balances were originally introduced in the geological sciences, and have come under several names, such as sequential binary partitions, or CoDa dendrograms.  Here we will refer to the visualizations representing balances as Balance Trees.\n",
```

Comment:
I've moved the dendrogram reference below - I don't think dendrograms should be mentioned until the tress have been introduced.  You think that is ok?